gplearn/genetic.py

```diff
@@ -203,6 +208,21 @@ def __init__(self,
         self.n_jobs = n_jobs
         self.verbose = verbose
         self.random_state = random_state
+        self.slim = slim
+        self._parent_generation = None
+        self._trained_generations = 0
```

Comment:
see my (pending) PR comments, I think some of these parameters can be discarded by doing things a slightly different way